leveldb_os/expiry_os_test.cc

```diff
@@ -1252,7 +1253,7 @@ TEST(ExpiryManifestTester, Overlap2)
 
     // let multiple threads complete
     /// sleep(1) required for Smart OS 1.8 buildbot
-    sleep(1);
+    sleep(2);
```

Comment:
What is the significance of changing the sleep() parameter from 1 to 2? Was this just a little extra slop to accommodate the buildbot? Regardless, the comment should be updated to match the code (or vice versa).